src/activities/content/ContentEntity.js

```diff
@@ -12,15 +14,17 @@ export const CONTENT_TYPES = {
 export class ContentEntity extends Entity {
 
 	/**
-	 * @returns {string} content-module link
+	 * @returns {string} content type
 	 */
 	getEntityType() {
 		// TODO - add more specific entity types to the activity-usage class array
-		// once we start working on non-module content types
+		// as we start working on new content types
 		if (!this._entity) {
 			return null;
-		} else if (this._entity.hasClass(CONTENT_TYPES.module)) {
+		} else if (this._entity.hasLinkByRel(Rels.Content.moduleEntity)) {
 			return CONTENT_TYPES.module;
+		} else if (this._entity.hasLinkByRel(Rels.Content.weblinkEntity)) {
```

Comment:
@CodeBaboon I'm pretty sure it's fine but wanted to double check, with [adding the weblink class](https://github.com/Brightspace/lms/pull/5172), this is the way to go?

Reply:
I think this is fine. I'm not sure if we have something like `hasLinkByClass` or not in this SDK. If we do, we could switch to it, but the by rel is fine also. Basically adding the `weblink` class felt like the 'right' thing to do, as well as allows for some potentially better generic code down the line, but wasn't really intended to impact this code in the here-and-now.